src/DonationForms/FormDesigns/ClassicFormDesign/css/_receipt.scss

```diff
@@ -1,324 +0,0 @@
-@use 'fn';
-
-.givewp-donation-confirmation-receipt {
-    width: 100%;
-    max-width: 100%;
-}
-
-.givewp-layouts-receipt {
-    border-radius: 0.5rem;
-    max-width: min(100%,51.5rem);
-    margin: 0 auto;
-    background: #fff;
-    box-shadow: 0 0.125rem 0.3125rem rgba(0, 0, 0, 0.3);
-
-    @media screen and (min-width: $desktopMinWidth) {
-        border-radius: 0.5rem;
-    }
-
-    .receipt-heading {
```

Comment:
Are the receipt styles being added to base styles?  I noticed it looks a little off, probably some small tweaks:

<img width="875" alt="Screenshot 2023-08-31 at 2 51 04 PM" src="https://github.com/impress-org/givewp/assets/10138447/b806dae5-6b57-40e1-8333-ef57d4231722">